runtime/README.md

```diff
@@ -227,6 +229,64 @@ Table: `INVOKE_DPE_COMMAND` output arguments
 | chksum      | u32           | Checksum over other output arguments, computed by Caliptra. Little endian.
 | data        | u8[...]       | DPE response structure as defined in the DPE iRoT profile.
 
+### QUOTE\_PCRS
+
+Generate a signed quote over all Caliptra hardware PCRs using the Caliptra PCR quoting key.
```

Comment:
How does one obtain this PCR quoting (public) key when using the mailbox?

Reply:
I believe this will be the FMC Alias Key. @ericeilertson @mhatrevi to confirm